frontend/src/components/MainDashboard/DashboardBody/DashboardBody.js

```diff
@@ -38,74 +38,143 @@ export default class DashboardBody extends Component {
   constructor(props) {
     super(props);
     this.state = {
-			data: [],
-			zipcode: '',
+      data: [],
+      zipcode: ""
     };
     this.getPet = this.getPet.bind(this);
     this.getPetByZipcode = this.getPetByZipcode.bind(this);
   }
 
-	componentDidMount() {
-		this.getPet();
-	}
+  handleChangeYoungToOld = () => {
+    if (this.state.zipcode === "") {
+      console.log("Success getting ordered from young to oldest");
+
+      //if there is no zipcode
+      fetch("/api/pet/asc", {
+        method: "GET",
+        headers: {
+          "Content-Type": "application/json; charset=utf-8"
+        }
+      })
+        .then(response => {
+          console.log(response.status);
+          if (response.status === 200) {
+            return response.json();
+          } else {
+            console.log("Something went wrong");
+          }
+        })
+        .then(jsonData => {
+          console.log(jsonData);
+          this.setState({ data: jsonData });
+        })
+        .catch(error => {
+          console.log(error);
+        });
+    } else {
+      //if there is zipcode
```

Comment:
What happens if there is a zipcode? This functionality might be better handled in the front since `getPets` and `getPetsByZipcode` returns a list of pets and can be sorted through a handler function. This way we can choose **ascending** and **descending** sorting even if `zipcode` exists.